src/JSInterop/Microsoft.JSInterop/test/JSRuntimeTest.cs

```diff
@@ -315,6 +315,90 @@ public void CanSanitizeDotNetInteropExceptions()
             Assert.Equal(invocation, error.InvocationInfo);
         }
 
+        [Fact]
+        public void ReceiveByteArray_AddsInitialByteArray()
+        {
+            // Arrange
+            var runtime = new TestJSRuntime();
+            runtime.DefaultTimeout = TimeSpan.FromSeconds(1);
```

Comment:
Instead of doing this on every test, why not set it as the default value on the property. Other tests don't seem to be setting this.

Also you likely can do, which is normally preferred
```csharp
            var runtime = new TestJSRuntime()
            {                                     
                DefaultTimeout = TimeSpan.FromSeconds(1)
            };
```

Reply:
Can probably just be removed, will take a look. Had copied from https://github.com/dotnet/aspnetcore/blob/1780543aa4ad0ba41dd3bc8253a8fb23ff611840/src/JSInterop/Microsoft.JSInterop/test/JSRuntimeTest.cs#L49.